### ISSUE TITLE
accuracy

### ISSUE THREAD

**Issue by mortezardana** [chandrikadeb7/Face-Mask-Detection#46]
Hi all,
I tried to test this model with different sizes of dataset. What I've done is to randomly select 1, 5, 10 or 20 % of the original dataset and training the model with train_mask_detector.py, but the accuracy value that is shown in the end of training is very odd. these are my results after training the model three times ( because I thought these numbers are wrong and wanted to make sure it gave me same numbers consistent before asking here ) : 

first time : 
10 % : 
      0.974025974025974
20 % : 
      0.974025974025974
_________________________
second time : 
10 % : 
     0.987012987012987
20 % : 
     0.9675324675324676
_________________________
third time : 
10 % : 
     0.974025974025974
20 % : 
     0.961038961038961

and one time when I was training with 5% it gave me the accuracy of 1.0
also called the sklearn.metrics.acuuracy_score which is the same function that is called after training under "[INFO] evaluating network..."
can anyone help me understand what I'm doing wrong or something??
thank you all

**Comment by Frightera** [comment]
Because your neural network is initialized with random values of weights, it means that then you'd have different starting points per each simulation during the training phase.
